### PR TITLE
Add ldrobot-lidar-ros2 entry

### PR DESCRIPTION
**Pull request by Myzhar**
Please add the following dependency to the rosdep database.

## Package name:

TODO

## Package Upstream Source:

TODO link to source repository

## Purpose of using this:

TODO Replace. This dependency is being used for this reason. This is why I think it's valuable to be added to the rosdep database.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
- Fedora: https://packages.fedoraproject.org/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL
- openSUSE: https://software.opensuse.org/package/
  - IF AVAILABLE

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME

# The source is here:

http://sourcecode_url

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: humble/distribution.yaml [hunk 1]
```diff
@@ -2161,6 +2161,25 @@ repositories:
...
+      - ldlidar_node
+      - ldlidar_component
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros2-gbp/ldrobot-lidar-ros2.git
+      version: 0.2.0
+    source:
+      type: git
+      url: https://github.com/Myzhar/ldrobot-lidar-ros2.git
+      version: humble
```
Comment (hidmic) [inline humble/distribution.yaml:2181]: @Myzhar the `humble` branch doesn't seem to exist in the source (and doc) repository.

File: humble/distribution.yaml [hunk 2]
```diff
@@ -2161,6 +2161,25 @@ repositories:
...
       version: humble
     status: maintained
+  ldrobot-lidar-ros2:
+    doc:
+      type: git
+      url: https://github.com/Myzhar/ldrobot-lidar-ros2.git
+      version: humble
+    release:
+      packages:
+      - ldlidar
```
Comment (hidmic) [inline humble/distribution.yaml:2171]: @Myzhar hmm, I see an MIT license mentioned in this package while everything points to Apache License, Version 2.0. Which one is it? Is it mixed?

**Review by hidmic — CHANGES_REQUESTED** [review]
@Myzhar was this `bloom` released? We strongly encourage [package releases](https://docs.ros.org/en/humble/How-To-Guides/Releasing/Releasing-a-Package.html) to be done using `bloom` and not manually.

File: humble/distribution.yaml (hunk 1 above)
Comment (Myzhar) [reply humble/distribution.yaml:2181]: Adding it

File: humble/distribution.yaml (hunk 2 above)
Comment (Myzhar) [reply humble/distribution.yaml:2171]: It was initially MIT, but then I changed to Apache-2. I search for remaining mentions to MIT

File: humble/distribution.yaml (hunk 1 above)
Comment (Myzhar) [reply humble/distribution.yaml:2181]: Fixed

File: humble/distribution.yaml (hunk 2 above)
Comment (Myzhar) [reply humble/distribution.yaml:2171]: Fixed

**Comment by hidmic** [comment]
@Myzhar see https://docs.ros.org/en/humble/How-To-Guides/Releasing/First-Time-Release.html

**Comment by clalancette** [comment]
Since this was not created with bloom, it almost certainly will not work.  I'll encourage you to follow the full instructions at https://docs.ros.org/en/humble/How-To-Guides/Releasing/First-Time-Release.html , and then let bloom open a new PR.

With that said, I'm going to close this out.

**Comment by nuclearsandwich** [comment]
@Myzhar thanks for your efforts with the process thus far, there are definitely some rough edges in the initial setup that we're working to both better document and smooth over.

For this first PR, which you correctly created manually, we should leave off the`release` section entirely. That will be added by bloom down the line after the release repository is configured and bloom can be used to create release data.

I would suggest updating this PR (or creating a new one and linking this discussion for context) which only adds the `source` and `doc` sections for your repository. Once that is merged your package will be "indexed" and will have also undergone the naming review required for the release repository setup in https://github.com/ros2-gbp/ros2-gbp-github-org/issues/159

Let me know if you have any other questions.

**Pull request closed without merging**